Packs/HealthCheck/ReleaseNotes/3_0_0.md

```diff
@@ -0,0 +1,129 @@
+
+#### Playbooks
+
+##### HealthCheck
+
+Updated the main playbook to differentiate between Cortex XSOAR versions 6.x and 8.x.
```

Comment:
Hi @tsachzim 
Usually, if we have the same playbook that works differently for v6 and v8, we will just create 2 versions of the playbook and will associate each one to the relevant platform version using the "marketplaces" key in the playbook file, for example, for the playbook which we want to run only on v6:
_**marketplaces: ["xsoar_on_prem"]**_

And for v8:
_**marketplaces: ["xsoar_saas"]**_

This makes the playbook more readable and I strongly advise to do so.
Since this is a community pack, I can approve this with its current structure (same playbook for both versions) but I strongly advise to change it to use 2 playbooks as I suggested above (even after this PR is merged in a different PR as an enhancement) since this pack is used by a lot of customers and published by XSOAR's CS team.
Also, I assume you did this already, but I strongly recommend to verify the playbook works on both versions.